de.metas.marketing/base/src/main/java/de/metas/marketing/base/interceptor/MKTG_ContactPerson.java

```diff
@@ -75,4 +97,45 @@ public void deleteCampaignContactPersonAdUserId(@NonNull final I_MKTG_ContactPer
 				.addEqualsFilter(I_MKTG_Campaign_ContactPerson.COLUMN_MKTG_ContactPerson_ID, contactPerson.getMKTG_ContactPerson_ID())
 				.create();
 	}
+
+	@CalloutMethod(columnNames = { I_MKTG_ContactPerson.COLUMNNAME_EMail })
+	public void onChangeEmail(final I_MKTG_ContactPerson contactPersonRecord, final ICalloutField field)
+	{
+
+		final ContactPersonRepository contactPersonRepo = Adempiere.getBean(ContactPersonRepository.class);
```

Comment:
you are within a spring `@Component`so there is no need to use `Adempiere.getBean()`

Reply:
agree.. u could declare it in constructor and it would be automatically autowired.

Reply:
done